src/library_manager.py

```diff
@@ -4434,12 +4492,25 @@ def mover_bound_volume(self, mticket):
         if self.use_threads:
             if not self.in_progress_lock.acquire(False):
                 Trace.trace(5, "mover_bound_volume: mover request in progress sending nowork %s"%(nowork,))
-                self.reply_to_caller(nowork)
+                threads = threading.enumerate()
+                for thread in threads:
```

Comment:
there is loop over threads just to be able to call Trace which is normally suppressed anyway. Is this really necessary?

Reply:
That is why I asked "Is this really necessary?". And you give ambiguous answer. It is either necessary or not. Is this kind of debug that was needed during development and testing of the code or the kind of debug logging you need to have when debugging issue on the live production system frequently? The answer to this question determines if this is necessary or not (IMO).